releasenotes/notes/computeuncompute-local-fidelity-501fe175762b7593.yaml

```
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Adds a flag `local` to the :class:`~.ComputeUncompute` state fidelity class that 
```

I think a single "tick" is only to annotate a Sphinx label that is usually used with eg. `:class:` or `:math:` -- for simple code snippets a double "tick" is the right annotation (afaik) 
```suggestion
    Adds a flag ``local`` to the :class:`~.ComputeUncompute` state fidelity class that 
```